### DISCUSSION TITLE
Charging for ATMs

### DISCUSSION THREAD

**Discussion by csm10495** [OpenRCT2/OpenRCT2#17421] [Feature requests] [answered]
Hey folks,

Should we make it possible to charge for ATMs? 

I know the original game didn't allow that, but in the real world, at least where I live, theme parks tend to charge for ATMs (either a flat fee and/or a percentage). A percentage would be a whole different paradigm, for the game, so instead maybe we should make it so they default to free, but could cost some amount of money, like a bathroom.

**Comment by James103** [comment]
As far as I can tell, IRL, when withdrawing money from an ATM, the fee charged by the ATM is subtracted from the bank account whose money was being withdrawn, in addition to the dollar amount requested. Therefore, the amount that would get added to the guest's purse remains the same regardless of the fee charged, but higher-fee ATMs should be less popular (i.e. fewer guests will use them, meaning more guests may leave the park due to running out of cash in pocket).

Therefore, the mechanics behind charging for ATMs in the game should boil down to this, from a player perspective: Would you rather have more money now when a guest gets money at the cost of guests leaving earlier on average due to low cash, or more money later since guests can withdraw from your ATMs for free and keep the money to spend in your park?

**Reply by csm10495** [reply]
I'm pretty sure some ATMs charge a usage fee that goes to the 'owner' of the ATM as opposed to a given bank. My thought is instead of pulling out X amount of money, they would pull X - 'the cost' with 'the cost' going into profits for the park owner.

**Comment by 733737** [comment]
Instead of charging a fee, you could just change the amount of money in guests pocket to something slightly less.

**Reply by csm10495** [reply]
The idea here would make the park owner money. Doing this particular suggestion wouldn't do that.

**Comment by 733737** [comment]
lowering the happiness of the guest is also an option

**Reply by csm10495** [reply]
This would be an interesting dynamic: higher the fee the larger the hit to happiness.

**Comment by AaronVanGeffen** [comment, marked as answer]
Recently, #26570 attempted to implement this feature request. We decided against it for several reasons, including how it disproportionally affects game balance.
